development/mock-dev.js

```diff
@@ -9,7 +9,7 @@
  * This is a convenient way to develop and test the plugin
  * without having to re-open the plugin or even re-build it.
  *
- * To use, run `npm run mock`.
+ * To use, run `yarn mock`.
```

Comment:
This command doesn't exist anymore 🤔 

It looks like it was removed in this commit: https://github.com/MetaMask/metamask-extension/commit/b6789a6e2fd9168e0bf8d386611d4c439ccd9e18

I'll put up a separate PR to remove this file as well, given that it can presumably no longer be used.